package.json

```diff
@@ -4,7 +4,6 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "pretest": "gulp lint",
     "test": "nyc ava"
```

Comment:
The `npm scripts` are usefull for precommit or prepush hooks. if you just want to run the tests you can use the command `ava`. because with `npm test` you also run `nyc`